src/mainwindow.cpp

```diff
@@ -313,6 +313,12 @@ MainWindow::MainWindow(Settings &settings
 
   setupModList();
   ui->espList->setup(m_OrganizerCore, this, ui);
+  connect(m_OrganizerCore.modList(), &ModList::modPrioritiesChanged, [this]() {
```

Comment:
I don't think we typically update the Data tab automatically, do we? There should be plenty of other places where we don't update it, so I'm not sure this is worth it.

Reply:
Hmm, seems you're right. The datatab is only updated if it's the currently visible tab when the directory structure is refreshed, which does not include mod removal for instance. I'll see if I can make it more consistent or I'll remove it from the PR.

Reply:
The data tab used to reset completely when refreshing iirc, and it was slow as heck, which is probably why it wasn't refreshing automatically. With the stuff I did, it might be possible to keep it up to date now. Maybe.

Reply:
I will time the `updateTree` (if it's not already) and check with some of my mod setup. What I think I'll do (if I manage) is to only refresh when the tab is visible, and otherwise set some kind of flag to refresh when the tab is made visible so that it should have no impact if you have another tab visible.

Reply:
There's already stuff in `MainWindow::on_tabWidget_currentChanged()` that calls `DataTab::activated()`, you can probably reuse it.